webhooks-extension/README.md

```diff
@@ -51,20 +53,76 @@ Access the Dashboard through its ClusterIP Service by running `kubectl proxy`. A
 
 Navigate to `Webhooks`, listed in the navigation under `Extensions`
 
-## Uninstall
 
-`kubectl delete -f config/release/gcr-tekton-webhooks-extension.yaml`
+## Notes On Using The Webhooks Extension
+
+Further to the limitations listed further below, it is worth noting that the webhooks extension does not currently work with all pipelines, it very specifically creates the following when the webhook is triggered:
+
+#### Git PipelineResource
+
+A PipelineResource of type git is created with:
+
+  - `revision` set to the short commit id from the webhook payload.
+  - `url` set to the repository url from the webhook payload.
+
+#### Image PipelineResource
+
+A PipelineResource of type image is created with:
+
+  - `url` set to ${REGISTRY}/${REPOSITORY-NAME}:${SHORT-COMMITID} where, REGISTRY is the value set when creating the webhook, other values are taken from the webhook payload.
+
+#### A PipelineRun
+
+A PipelineRun for your chosen pipeline; in the namespace specified when your webhook was created; the values assigned to parameters on the pipelinerun are taken from values provided when configuring the webhook or from the webhook payload itself.
```

Comment:
Nit but I'd use commas instead of so many semicolons 😄

Reply:
changed to commas